geniza/corpus/templates/corpus/document_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static i18n %}
+
+{% comment %}Translators: Title for document search page{% endcomment %}
+{% translate 'Search Documents' as doctitle%}
+
+{% block title %}{{ doctitle }}{% endblock title %}
+
+{% block main %}
+<div class="container search">
+    <h1>{{ doctitle }}</h1>
+
+    <form>
+    {{ form }}
+    {% for field, errors in form.errors.items %}
+    {# no clean way to get field labels here, so we just collapse the error lists #}
+        {% for error in errors %}
+            <li>{{ error }}</li>
+        {% endfor %}
+    {% endfor %}
+    </form>
+
+    {% comment %}Translators: number of search results{% endcomment %}
+    {% blocktranslate count counter=total %}
+        1 result
+    {% plural %}
+        {{ counter }} total results
+    {% endblocktranslate %}
+
+    <ol>
+    {% for document in documents %}
+        <li class="result">
+
+            <dl class="tags">
+                <dt class="sr-only">{% translate 'Tags' %}</dt>
+                {% for tag in document.tags %}
+                <dd class="tag">#{{ tag }}</dd>
+                {% endfor %}
+            </dl>
```

Comment:
it's still a bit weird to me that these go here. semantically, the heading should be the first thing inside its parent element (the `<li>`). we could use CSS to move the tags above the record, but that feels a little weird too.

Reply:
I agree — I struggled a little with the counter placement too, until I figured out I could add it relative to the h1. Ok for a first implementation though, right?

Reply:
yes, for sure – and nice work on the counter placement!